SPECS/libstoragemgmt/libstoragemgmt.spec

```diff
@@ -0,0 +1,911 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+
+%bcond_with     test
```

Comment:
Remove this as well when you remove the %if test around the %check section

Reply:
Done.